### PR TITLE
Changelogs for Noetic release 0.11.1

### PR DESCRIPTION
**Pull request by tylerjw**
One small change, this next release should be really easy.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,11 @@
 Changelog for package moveit_msgs
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+0.11.1 (2020-10-09)
+-------------------
+* [documentation] add a disclaimer to CO about pose not working yet (`#90 <https://github.com/ros-planning/moveit_msgs/issues/90>`_)
```
Comment (felixvd) [inline CHANGELOG.rst:7]:
```suggestion
* [documentation] add disclaimer to CO about object pose not working yet (`#90 <https://github.com/ros-planning/moveit_msgs/issues/90>`_)
```

**Review by felixvd — APPROVED** [review]
Just a clarification, otherwise good with me

**Review by rhaschke — COMMENTED** [review]
@felixvd, what's the status of the corresponding PR to get the `pose` field actually used?
Shouldn't we try to merge this before next release?

**Comment by felixvd** [comment]
It's just waiting on reviews. Given the size, I doubt that it will be ready for this release, so I approved this as is.

**Pull request merged**